interface/forms/group_attendance/new.php

```diff
@@ -29,9 +29,15 @@
 require_once("functions.php");
 require_once(dirname(__FILE__) . "/../../../library/group.inc");
 
-//Check acl if current user can edit form
+//Check acl
+$can_view = acl_check("groups","gadd",false, 'view');
 $can_edit = acl_check("groups","gadd",false, 'write');
 
+if (!$can_view && !$can_edit)
```

Comment:
This would have been better written as `if ($can_view === false && $can_edit === false)` this is easier to read and better describes what's actually taking place.

Also, PSR requires camel case variables for methods/classes and would recommend the same for this type of variables (i.e. `$canView`)